main.go

```diff
@@ -123,6 +136,12 @@ func add(c config.Config, path, name string) {
 }
 
 func rm(c config.Config, path, name string) {
+	_, err := c.Get(name)
+	if err != nil {
+		fmt.Println("It's not posible remove "+name+" as it doesn't exist.")
```

Comment:
I would change the text by `Alias not found` instead.

Reply:
Great. I've just done it.